### ISSUE TITLE
Defect-level-4 [SCREENREADER]: Update markup in DD bank info Additional Info

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#22522]
taken from [Josh Kim's comments here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19611#issuecomment-811949304):

Adding `<figure>` is the right way forwards, but it also means we need to add in some minor changes to ensure it has better coverage for assistive technology and so that it will be less verbose. I realized this after testing again (see video below).

These are defect-level-4s, but if it's a quick fix it would make the figure much easier to navigate 😄 

### Original
```html
<figure>
  <img src="/img/direct-deposit-check-guide.svg" 
    role="img" 
    alt="A personal check" 
    aria-labelledby="check-caption">
  <figcaption id="check-caption">
    <p>The bank routing number is the first 9 digits on the bottom left corner of a printed check. Your account number is the second set of numbers on the bottom of a check, just to the right of the bank routing number.</p><p>If you don’t have a printed check, you can:</p>
     <ul>
      <li>Sign in to your online bank account and check your account details, or</li>
      <li>Check your bank statement, or</li><li>Call your bank</li>
    </ul>
  </figcaption>
</figure>
```

https://user-images.githubusercontent.com/14154792/113309671-6085c180-92d5-11eb-932a-239ebe979247.mov

In this video, the caption is repeated three times in a row.

### Proposed

```diff
+ <!-- Add role="figure" and aria-labelledby="check-caption" to the <figure> for better AT coverage -->
+ <figure role="figure" aria-labelledby="check-caption">
  <img src="/img/direct-deposit-check-guide.svg" 
    role="img" 
    alt="A personal check" 
- <!-- Remove aria-labelledby from the image to reduce verbosity as the caption is covered by use of figure -->
-    aria-labelledby="check-caption">
  <figcaption id="check-caption">
    <p>The bank routing number is the first 9 digits on the bottom left corner of a printed check. Your account number is the second set of numbers on the bottom of a check, just to the right of the bank routing number.</p><p>If you don’t have a printed check, you can:</p>
     <ul>
      <li>Sign in to your online bank account and check your account details, or</li>
      <li>Check your bank statement, or</li><li>Call your bank</li>
    </ul>
  </figcaption>
</figure>
```

## Acceptance Criteria
- [x] Execute as outlined above 
- [x] When ready for validation ask Josh Kim or Angela Fowler for their review

**Comment by ajakabcin** [comment]
hey @joshkimux or @AngelaFowler82 could one you take a look at this for us? Thank you!

**Comment by joshkimux** [comment]
@ajakabcin this should be good to go 🥳
